test/utils/bytecode.hpp

```diff
@@ -90,30 +91,79 @@ big_endian(T value)
     return {static_cast<uint8_t>(value >> 8), static_cast<uint8_t>(value)};
 }
 
-inline bytecode eof_header(
-    uint8_t version, uint16_t code_size, uint16_t max_stack_height, uint16_t data_size)
+struct eof_bytecode
```

Comment:
I don't mind leaving it as is because we are not consistent in any way, but you can rename it to CamelCase if you prefer.

Reply:
I would leave it for consistency with `call_instruction`, `create_instruction`.